src/assets/js/react/components/FontManager/AddFontFooter.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Spinner from '../Spinner'
+
+const AddFontFooter = ({
+  id,
+  onHandleCancelEditFont,
+  msg: { success, error },
+  loading,
+  tabIndexFooterButtons
+}) => {
+  const cancelButton = document.querySelector('.footer button.cancel')
+
+  return (
+    <div className={'footer' + (cancelButton ? ' cancel' : '')}>
+      {id && (
+        <button
+          className='button gfpdf-button primary cancel'
+          onClick={onHandleCancelEditFont}
+          tabIndex={tabIndexFooterButtons}
+        >
+          ← Cancel
+        </button>
+      )}
+
+      <button
```

Comment:
I think we need to disable the Update Font button if none of the state has changed. It's a bit odd clicking the button and nothing happens unless you've modified the form inputs.